app/controllers/imports/issues_controller.rb

```diff
@@ -38,11 +38,10 @@ def set_import_issues
       )
 
     @patients =
-      policy_scope(Patient).with_pending_changes.includes(
-        :gp_practice,
-        :school,
-        :school_moves
-      )
+      policy_scope(Patient)
+        .with_pending_changes
```

Comment:
It feels like we'll often need to combine `with_pending_changes` and `not_archived`. I wonder if it's worth creating a scope which captures this combination that we can use everywhere, something like:

```rb
scope :with_pending_changes_for_team, ->(team) do
  with_pending_changes.not_archived(team:)
end
```